lib/speedy_af/base.rb

```diff
@@ -92,7 +92,10 @@ def gather_reflections(proxy_hash, opts)
 
       def query_for_belongs_to(proxy_hash, _opts)
         proxy_hash.collect do |_id, proxy|
-          proxy.belongs_to_reflections.collect { |_name, reflection| "id:#{proxy.attrs[predicate_for_reflection(reflection).to_sym]}" }
+          proxy.belongs_to_reflections.collect do |_name, reflection|
+            next if proxy.attrs[predicate_for_reflection(reflection).to_sym].blank?
+            "id:#{proxy.attrs[predicate_for_reflection(reflection).to_sym]}"
```

Comment:
How about introducing a variable to avoid running this chain twice?
```suggestion
            id = proxy.attrs[predicate_for_reflection(reflection).to_sym]
            id.blank? ? nil : "id:#{id}"
```